save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/frontend/PlatformLanguages.kt

```diff
@@ -28,5 +30,11 @@ enum class PlatformLanguages(val code: String, val value: String, val label: Str
          * Default platform language
          */
         val defaultLanguage = EN
+
+        /**
+         * @param code language code
+         * @return [PlatformLanguages] enum entity corresponding to language [code]
+         */
+        fun getByCode(code: String) = values().find { it.code == code }
```

Comment:
maybe `getByCodeOrDefault` instead of returning nullable result

Reply:
Ok, will do it

Reply:
Done